bin/fusioninventory-injector

```diff
@@ -125,6 +131,9 @@ sub sendContent {
         print $res->is_success() ?
             "OK\n" : "ERROR: " . $res->status_line() . "\n";
     }
+    unless ($res->is_success()) {
+        die "Content has not been sent. HTTP status is : " . $res->status_line;
```

Comment:
Finally I disagree to add a `die` here as you're breaking the script behavior: if someone wants to inject a lot of inventory, some could not be injected after the first error.